### DISCUSSION TITLE
子应用资源加载失败，需要怎么配置呢

### DISCUSSION THREAD

**Discussion by llbiao** [jd-opensource/micro-app#95] [Q&A]
![image](https://user-images.githubusercontent.com/32304163/133870565-94e04ba2-540a-4ad4-8436-04129741aad0.png)
![image](https://user-images.githubusercontent.com/32304163/133870584-2fdf10b6-0355-472b-ae8a-26fa74bd350c.png)

**Comment by bailicangdu** [comment]
可以提供一个demo用来复现吗

**Reply by llbiao** [reply]
这应该是子应用资源跨域了，请问需要怎么处理跨域问题呢？

**Comment by bailicangdu** [comment]
开发环境可以配置webpack，线上环境可以配置nginx

**Comment by bailicangdu** [comment]
参考这里 https://micro-zoe.github.io/micro-app/docs.html#/zh-cn/questions?id=_2%e3%80%81%e5%ad%90%e5%ba%94%e7%94%a8%e4%b8%80%e5%ae%9a%e8%a6%81%e6%94%af%e6%8c%81%e8%b7%a8%e5%9f%9f%e5%90%97%ef%bc%9f

**Reply by llbiao** [reply]
已配置但仍然跨域，在官方提供的deom上试过也不行；
你可以试下如两个链接：
1、https://csc.cvte.com/yc-ccbar/ccbar/ccbar_sdk.js
2、https://hm.baidu.com/hm.js?19784915071ffe84617a27a62ccd2173
![image](https://user-images.githubusercontent.com/32304163/133892222-3bbfd032-2dd2-46fd-8505-bc2ae23068dd.png)
![image](https://user-images.githubusercontent.com/32304163/133892229-4c69e96b-34f7-4681-bc11-381806a01221.png)

**Reply by bailicangdu** [reply]
这个地址都没支持跨域

**Reply by llbiao** [reply]
那有办法解决吗

**Reply by bailicangdu** [reply]
把这2个js放在基座应用里
